src/component/MintNft/MintNftForm/MintNftForm.jsx

```diff
@@ -76,17 +77,64 @@ export default function MintNftForm({ setToastMessage }) {
     return dataURI;
   }
 
+  let imgHash;
+  const sendFileToIPFS = async (file) => {
+    if (file) {
+      try {
+        const formData = new FormData();
+        formData.append('file', file[0]);
+
+        let apiKey = '98cbaf5b71172582997a';
```

Comment:
Api keys and secrets redefined the same in both the functions. Keep them at the top level